hta/trace_analysis.py

```diff
@@ -520,15 +520,20 @@ def critical_path_analysis(
         This will include GPU kernels launched by the cpu operators in that
         time duration.
         For example, you can use this to limit the analysis to one iteration
-        by passing annotation='ProfilerStep500'. See notes for how to pick the iteration.
+        by passing annotation='ProfilerStep'. See notes for how to pick the iteration.
 
         Args:
             t (Trace): Input trace data structure.
             rank (int): rank to analyze for the critical path.
             annotation (str): a trace annotation to limit the analysis to,
-                such as "ProfilerStep500"
-            instance_id (int): optionally specify which instance of the annotation
-                to consider. Defaults to the first instance.
+                for example "ProfilerStep" would match all annotations that
+                match this string (ProfilerStep100, ProfilerStep101 etc)
```

Comment:
the exact annotation is like `ProfilerStep#100`. Does it matter if the pound sign is missing?

Reply:
@anupambhatnagar yes it doesn't need the '#' as such as it matches a substring, using "ProfilerStep" would work. But i will update the doc to add '#' to be correct